apps/ecs/ecs-cli/ecs-params-create.sh

```diff
@@ -30,6 +30,6 @@ fi
 perl -i -pe 's/sg-replaceme/'${SecurityGroupWebapp}'/g' $PARAM_FILE
 
 mv $PARAM_FILE $SERVICENAME/
-echo "Changing directory to $SERVICENAME \n"
-cd $SERVICENAME/
+#echo "Changing directory to $SERVICENAME \n"
+cd ./greeting
```

Comment:
Is this change required?

Reply:
In general, I don't like directories changing magically. Can we just stick to the file generation in service specific directory and update the CLI to access the file from that directory instead?